_episodes/00-before-we-start.md

```diff
@@ -201,7 +202,7 @@ messages that might not be very helpful to diagnose a problem (e.g. "subscript o
 the message is very generic, you might also include the name of the function or package you’re using
 in your query.
 
-However, you should check Stack Overflow. Search using the `python` tag. Most questions have already
+However, you should check Stack Overflow. Search using the `[python]` tag. Most questions have already
```

Comment:
Why square brackets here? Is it because https://stackoverflow.com/questions/tagged/python says "Questions tagged [python] " ?

Reply:
Yes. The monospace formatting IMHO implied an as accurate quote as possible.

Reply:
I'm just afraid people might search literally for [python] ...